src/coreclr/scripts/superpmi.py

```diff
@@ -278,6 +278,7 @@ def add_core_root_arguments(parser, build_type_default, build_type_help):
 superpmi_common_parser.add_argument("--break_on_error", action="store_true", help=break_on_error_help)
 superpmi_common_parser.add_argument("--skip_cleanup", action="store_true", help=skip_cleanup_help)
 superpmi_common_parser.add_argument("--sequential", action="store_true", help="Run SuperPMI in sequential mode. Default is to run in parallel for faster runs.")
+superpmi_common_parser.add_argument("--parallelism", help="Specify amount of parallelism")
```

Comment:
I always thought this option was available in `superpmi.py` or at least it passed this option to `superpmi.exe`. Weren't we doing this for `tpdiff`?

Reply:
It is, but there is no way to control the degree of parallelism. On my CPU it defaults to 32 workers, but 16 works better for me when doing tpdiff (it's 15-20% faster). This allows you to pass `--parallelism 16` and it will pass `-p 16` instead of just `-p` to superpmi.exe.

Reply:
Got it. So does it defaults to number of processors?

Reply:
Yes, superpmi.exe will use the logical processor count by default when `-p` is specified without a value.